gateway-types/src/ignition/v1.rs

```diff
@@ -135,6 +145,7 @@ impl From<gateway_messages::ignition::SystemType> for SpIgnitionSystemType {
             SystemType::Sidecar => Self::Sidecar,
             SystemType::Psc => Self::Psc,
             SystemType::Unknown(id) => Self::Unknown { id },
+            SystemType::Cosmo => Self::Unknown { id: 0x4 },
```

Comment:
Could you leave a comment explaining what `0x4` means, or where this ID comes from? If this is a value that could be used elsewhere, it'd be nice to have it as a constant as well.